### PR TITLE
Reducing E2E run time

### PR DESCRIPTION
**Pull request by sawood14012**
# Description

An attempt to reduce e2e time

- [X] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

**Comment by sawood14012** [comment]
this Commit Statistics 
`17 features passed, 0 failed, 6 skipped

570 scenarios passed, 0 failed, 116 skipped

6613 steps passed, 0 failed, 2855 skipped, 0 undefined

Took 22m25.243s`

File: integration-tests/feature_list.txt [hunk 1]
```diff
@@ -20,7 +20,7 @@ features/stack_analysis_pypi_direct_and_transitive.feature
 features/stack_analyses_maven_ecosystem.feature
 features/stack_analysis_maven_direct_and_transitive.feature
 features/stack_analyses_npm_ecosystem.feature
-features/stack_analysis_npm_direct_and_transitive.feature
+#features/stack_analysis_npm_direct_and_transitive.feature
```
Comment (sawood14012) [inline integration-tests/feature_list.txt:23]: Exact same tests are present in 
https://github.com/sawood14012/fabric8-analytics-common/blob/master/integration-tests/features/stack_analyses_smoke_tests_npm_ecosystem.feature

File: integration-tests/feature_list.txt [hunk 2]
```diff
@@ -37,8 +37,8 @@ features/three_scale.feature
 #features/api_backbone.feature
 features/gemini.feature
 features/resilient_infrastructure.feature
-features/three_scale_stack_analyses.feature
-features/three_scale_component_analyses.feature
+#features/three_scale_stack_analyses.feature
```
Comment (sawood14012) [inline integration-tests/feature_list.txt:40]: three scale tests not required as we run everything on three scale now

**Comment by sawood14012** [comment]
after some more changes 

> 17 features passed, 0 failed, 6 skipped
   570 scenarios passed, 0 failed, 116 skipped
   6590 steps passed, 0 failed, 2855 skipped, 0 undefined
   Took 20m49.834s

**Comment by sawood14012** [comment]
Previously the e2e was taking `37m46.099s`

**Comment by sawood14012** [comment]
Final round test stats :

`17 features passed, 0 failed, 6 skipped`
`570 scenarios passed, 0 failed, 116 skipped`
`6549 steps passed, 0 failed, 2855 skipped, 0 undefined`
`Took 18m56.715s`

File: integration-tests/feature_list.txt (hunk 2 above)
Comment (prashbnair) [reply integration-tests/feature_list.txt:40]: Can you also remove the code associated with this?

**Comment by sawood14012** [comment]
we can remove the feature file the code associated with this is being used in V1 tests

**Comment by sawood14012** [comment]
[test]

File: integration-tests/feature_list.txt [hunk 3]
```diff
@@ -20,8 +20,8 @@ features/stack_analysis_pypi_direct_and_transitive.feature
 features/stack_analyses_maven_ecosystem.feature
 features/stack_analysis_maven_direct_and_transitive.feature
 features/stack_analyses_npm_ecosystem.feature
-features/stack_analysis_npm_direct_and_transitive.feature
-features/stack_analyses_smoke_tests_pypi_ecosystem.feature
+#features/stack_analysis_npm_direct_and_transitive.feature
+#features/stack_analyses_smoke_tests_pypi_ecosystem.feature
```
Comment (sawood14012) [inline integration-tests/feature_list.txt:24]: ecosystem_pypi_pylist.feature and smoke_tests_pypi were same but one contains SLA/SLO checks other doesn't so merged them by adding SLA/SLO checks in the ecosystem feature and commented out the other feature

File: integration-tests/features/stack_analyses_pypi_ecosystem_pylist.feature [hunk 4]
```diff
@@ -32,6 +32,10 @@ Feature: Thorough stack analysis v3 API tests for PyPi ecosystem, with pylist.js
      And I should find the following dependencies (click) in the stack analysis
      And I should find dependency named click with version 6.7 in the stack analysis
 
+    # SLA/SLO-related checks
```
Comment (sawood14012) [inline integration-tests/features/stack_analyses_pypi_ecosystem_pylist.feature:35]: here are the SLA/SLO checks that have been added

**Comment by sawood14012** [comment]
[test]

**Comment by sawood14012** [comment]
[test]

**Pull request merged**